client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -147,8 +148,12 @@ const UpgradeNudge = React.createClass( {
 } );
 
 export default connect(
-	state => ( {
-		site: getSelectedSite( state ),
-	} ),
+	( state, ownProps ) => {
+		const siteId = getSelectedSiteId( state );
```

Comment:
could this be unset here? what if we navigate to the page URL without a site selected?